### PR TITLE
add workflow for bot approval

### PR DESCRIPTION
**Pull request by jdkandersson**
Applicable spec: ISD115

### Overview

Adds a workflow file that triggers `operator-workflows` for the purpose of providing 1 approval in case of a bot PR

### Rationale

To make it possible for 1 person to manage bot PRs rather than having to wait for an additional review

### Juju Events Changes

N/A

### Module Changes

N/A

### Library Changes

N/A

### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [x] The documentation for charmhub is updated.
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

File: .github/workflows/bot_pr_approval.yaml
Comment (jdkandersson) [inline .github/workflows/bot_pr_approval.yaml:8]: Not referencing main branch since still need to make it work and test it

**Review by yanksyoon — APPROVED** [review]

**Comment by jdkandersson** [comment]
Since this just adds a workflow file unrelated to any of the code and we're having issues with integration tests failing at the moment due to an issue in one of the actions I will merge this without tests passing when there are 2 approvals

**Pull request merged**
